src/databases/IDX/avtIDXFileFormat.C

```diff
@@ -1,6 +1,40 @@
-// Copyright (c) Lawrence Livermore National Security, LLC and other VisIt
-// Project developers.  See the top-level LICENSE file for dates and other
-// details.  No copyright assignment is required to contribute to VisIt.
+/*****************************************************************************
```

Comment:
Please revert the copyright notice back to the new shortened version.

Reply:
Done, thank you.